share/completions/rsync.fish

```diff
@@ -2,8 +2,24 @@ function __rsync_remote_target
     commandline -ct | string match -r '.*::?(?:.*/)?' | string unescape
 end
 
+function __rsync_parse_flags -d "Print info|help FLAGS from help output"
+    set -l helptext $argv[2..-1]
```

Comment:
If I'm reading this correctly, you're skipping the first line because it's a heading - the output looks like

```
Use OPT or OPT1 for level 1 output, OPT2 for level 2, etc.; OPT0 silences.

BACKUP     Mention files backed up
COPY       Mention files copied locally on the receiving side
DEL        Mention deletions on the receiving side
FLIST      Mention file-list receiving/sending (levels 1-2)
MISC       Mention miscellaneous information (levels 1-2)
MOUNT      Mention mounts that were found or skipped
NAME       Mention 1) updated file/dir names, 2) unchanged names
PROGRESS   Mention 1) per-file progress or 2) total transfer progress
REMOVE     Mention files removed on the sending side
SKIP       Mention files that are skipped due to options used
STATS      Mention statistics at end of run (levels 1-3)
SYMSAFE    Mention symlinks that are unsafe

ALL        Set all --info options (e.g. all4)
NONE       Silence all --info options (same as all0)
HELP       Output this help message
```

If so, there should be a comment that says so, for future readers.

Reply:
Yes, you read that correctly, I have added a comment.